cmd/operator/README.md

```diff
@@ -78,4 +79,13 @@ $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/
 ```
 
 Now you can deploy the device plugins by creating corresponding custom resources.
-The samples for them are available [here](/deployments/operator/samples/).
\ No newline at end of file
+The samples for them are available [here](/deployments/operator/samples/).
+
+## Known issues
+
+When the operator is run with leader election enabled, that is with the option
+`--enable-leader-election`, make sure the cluster is not overloaded with excessive
+number of pods. Otherwise a heart beat used by the leader election code may trigger
+a timeout and crash. We are going to use different clients for the controller and
+leader election code to alleviate the issue, but in case of an overloaded cluster
+the controller's client will crash anyway when it hits a timeout.
```

Comment:
This doesn't sound too optimistic to me. Should we give a reader at least some hope for a possible future fix?

Reply:
We can simply drop the last "but" to make it sound less morbid :)

Reply:
sgtm

Reply:
done